docs/source/configuration/multilingual.md

```diff
@@ -9,13 +9,14 @@ myst:
 
 # Multilingual
 
-Volto provide support for Plone's Multilingual feature. You need to install Multiligual
-support in Plone (`plone.app.multilingual` add-on), that comes available by default since
-Plone 5 and can be installed in Plone's control panel.
+Volto provides support for Plone's multilingual feature through the Plone core add-on `plone.app.multilingual`.
+It is included with Plone 5 and later.
+You can enable it through the Plone Classic UI administrative interface.
```

Comment:
Why did you change this to mention the Classic UI specifically? Add-ons can also be installed via the add-ons control panel in Volto.

Reply:
Done in b494e42aeecf689d111aac967beed0c6e0fc699d